conda/cli/install.py

```diff
@@ -64,7 +67,51 @@
 stderrlog = getLogger("conda.stderr")
 
 
-def check_prefix(prefix, json=False):
+def validate_prefix_exists(prefix: str | Path) -> None:
+    """
+    Validate that we are receiving at least one valid value for --name or --prefix.
+    """
+    prefix = Path(prefix)
+    if not prefix.exists():
+        raise CondaEnvException("The environment you have specified does not exist.")
+
+
+def check_protected_dirs(prefix: str | Path, json: bool = False) -> None:
+    """Ensure that the new prefix does not contain protected directories."""
+    history_file = Path(prefix).parent / "conda-meta" / "history"
+
+    if history_file.is_file():
```

Comment:
Hm the recent convos in https://github.com/conda/conda/pull/14112 reminds me that we probably want to use `conda.gateways.disk.tests.is_conda_environment` instead. Also begs the question whether these helpers should also be defined in `conda.gateways.disk.test`.